draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -1435,7 +1447,7 @@ uCDN-A                         dCDN-B                         dCDN-C
         that intermediate CDN MUST similarly transform URLs in triggers it passes to its dCDNs.</t>
    <t>
         When processing a trigger, CDNs MUST ignore the URL scheme (HTTP or HTTPS) in comparing URLs.
-        For example, for a CI/T "invalidate" or "purge" action, content MUST be invalidated or
+        For example, for a CI/T "invalidate" or "purge" action, content MUST be invalidated, or
```

Comment:
This added an unnecessary comma after invalidated.  I'm not sure what this accomplishes other than to make the sentence less readable.